themes/classic/templates/catalog/_partials/miniatures/product.tpl

```diff
@@ -82,6 +82,7 @@
         {/block}
       </div>
 
+      <!-- @todo: use include file='catalog/_partials/product-flags.tpl'} -->
```

Comment:
In order to reduce the number of changes so close to RC, I prefer to perform the refactoring in next version

Reply:
fair enough